src/boot/efi/assert.c

```diff
@@ -6,7 +6,7 @@
 #include "util.h"
 
 void efi_assert(const char *expr, const char *file, unsigned line, const char *function) {
-        log_error_stall(L"systemd-boot assertion '%a' failed at %a:%u, function %a(). Halting.", expr, file, line, function);
+        log_error_stall(L"systemd-boot assertion '%a' failed at %a:%" PRIuN L", function %a(). Halting.", expr, file, line, function);
```

Comment:
is this right?

reading the comments in `PrintLib.h` it appears to me %u is size of an int, and hence also size of an unsigned int, and hence the code actually was right here